src/VolumeCache.ts

```diff
@@ -30,14 +30,14 @@ type CacheEntry = {
   parentArr: CacheEntry[];
 };
 
-type CachedVolumeScale = {
+type CacheStoreScale = {
```

Comment:
suggested: `CacheStoreMultiscaleLevel`.  Pyramid is also a reasonable term that is used for Multiscale or Multiresolution.  Personally I prefer Multiresolution but any of them work fine